wp-content/themes/haruoshi_origin/functions.php

```diff
@@ -189,3 +189,55 @@ function taxonomy_writer() {
 ));
 
 endif;
+
+/**
+ * ページネーション
+ */
+function pagination($pages = '')
+{
+     global $paged; //現在のページ値
+     if(empty($paged)) $paged = 1; //デフォルトのページ
+
+     if($pages == '') {
+         global $wp_query;
+         $pages = $wp_query->max_num_pages;//全ページ数を取得
+         if(!$pages) { //全ページ数が空の場合は、１とする
+             $pages = 1;
+         }
+     }
+
+     if(1 != $pages) { //全ページが１でない場合はページネーションを表示する
+		echo "<div class=\"pager\">\n";
+		echo "<ul class=\"pagination\">\n";
```

Comment:
```suggestion
		echo '<ul class="pagination">';
```
なおす必要はないけどこうすると楽だしみやすいよ

Reply:
done https://github.com/oooshima/origin_wordpress/pull/35/commits/e4ffb8f73966eaf5582ae6b66a3bfafd09f49d22